src/core/extent_tree.cc

```diff
@@ -29,7 +29,7 @@ void ExtentTree::Add(size_t start, size_t len) {
       prev->second += len;
       start = prev->first;
       len += prev_len;
-      prev_extent = prev;
+      prev_extent_key = prev->first;
```

Comment:
absl also issued a warning but the test did not fail. Maybe we should somehow be aware of these warnings (because they usually point to the root cause of the problem)